hoodie-common/src/main/java/com/uber/hoodie/common/table/HoodieTableMetaClient.java

```diff
@@ -198,7 +201,7 @@ public static HoodieTableMetaClient initTableType(Configuration hadoopConf, Stri
     Properties properties = new Properties();
     properties.setProperty(HoodieTableConfig.HOODIE_TABLE_NAME_PROP_NAME, tableName);
     properties.setProperty(HoodieTableConfig.HOODIE_TABLE_TYPE_PROP_NAME, tableType.name());
-    if (tableType == HoodieTableType.MERGE_ON_READ) {
+    if (tableType == MERGE_ON_READ) {
```

Comment:
should we persist the index type into table properties.. I think it might be good to do that?